docs/docs/extend/plugins/barcode.md

```diff
@@ -23,3 +23,35 @@ POST {
     barcode_data: "[(>someBarcodeDataWhichThePluginKnowsHowToDealWith"
 }
 ```
+
+### Example
+Please find below a very simple example that is executed each time a barcode is scanned.
+
+```python
+from django.utils.translation import gettext_lazy as _
+
+from InvenTree.models import InvenTreeBarcodeMixin
+from plugin import InvenTreePlugin
+from plugin.mixins import BarcodeMixin
+
+class InvenTreeBarcodePlugin(BarcodeMixin, InvenTreePlugin):
+
+    NAME = "MyBarcode"
+    TITLE = "My Barcodes"
+    DESCRIPTION = "support for barcodes"
+    VERSION = "0.0.1"
+    AUTHOR = "Michael"
+
+    Status=0
```

Comment:
I would stick to conventions here and use lowercase snake case for variables. And secondly I would add some spacing around the =. (Also at other lines)
```suggestion
    status = 0
```

Reply:
@SergeoLacruz please address this as suggested by @wolflu05 then happy to merge